ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -129,6 +130,7 @@ export const GetStatusResponseSchema: yup.ObjectSchema<GetNodeStatusResponse> =
       .object({
         synced: yup.boolean().defined(),
         head: yup.string().defined(),
+        head_timestamp: yup.number().defined(),
```

Comment:
This convention is a totally new naming convention

Reply:
Yea sorry.. fixed this